testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/CookieStoreRootContextTest.java

```diff
@@ -54,15 +53,6 @@
 @AppServerContainer(ContainerConstants.APP_SERVER_EAP6)
 public class CookieStoreRootContextTest extends DemoServletsAdapterTest {
```

Comment:
imho `CookieStoreRootContextTest` shouldn't inherit from `DemoServletsAdapterTest` but rather from `AbstractServletsAdapterTest` 

In this state it runs all tests from `DemoServletsAdapterTest` twice.

Reply:
That's true. However, I think it's intentional (or partially intentional).

The `testTokenInCookieSSORoot` uses `customerPortal` variable, which is defined in `DemoServletsAdapterTest`. Also, `loginToCustomerCookiePortalRoot` uses `assertLogged`, which again, is defined in `DemoServletsAdapterTest`. 

So, I guess we can either leave this like this or move `customer-cookie-portal-root` deployment into `DemoServletsAdapterTest` and simply remove `CookieStoreRootContextTest`. 

WDYT?

Reply:
The latter option seems better to me. I'd go with this one.

Reply:
Ok, I'll try it out tomorrow and push an update. Thanks for the feedback.

Reply:
Implemented